csm/CameraMan.py

```diff
@@ -0,0 +1,48 @@
+<<<<<<< HEAD
```

Comment:
I don't know why these were marked as merge conflicts but can you get rid of all of these lines?

Reply:
👍